preview/src/index.html

```diff
@@ -36,23 +48,24 @@
         const metadata = payload.metadata;
 
         return {
+            audience: getAudience(metadata?.previewMode, metadata?.audienceName),
```

Comment:
```suggestion
        const metadata = payload.metadata;
        const audience = metadata?.previewMode === 'slotDefaultContent' ? 'None' : metadata?.audienceName ?? 'Unknown';

        return {
            audience: audience,
```

Reply:
Actually I have divided thoughts about this suggestion, even though we can remove that function, this ternary is not very readable, which make me think that the function approach is better. I'll leave the suggestion here for you to decide

Reply:
I decided by function for readability, but thanks for the suggestion.